packages/storybook/stories/icon/icon-gallery.stories.mdx

```diff
@@ -1,6 +1,7 @@
 <!-- @license CC0-1.0 -->
 
 import { Canvas, IconGallery, IconItem, Meta } from "@storybook/addon-docs";
+import iconSet from "@utrecht/icon/tmp/components/icon-set.json";
```

Comment:
Do we now need extra instruction in our readme to do `npm run build` before `npm run storybook`, or is generating this all part of the `npm run (pre)storybook` build steps?